src/test/java/jp/co/future/uroborosql/parameter/mapper/BindParameterMapperManagerTest.java

```diff
@@ -66,6 +71,23 @@ public void test() throws ParseException {
 		assertThat(parameterMapperManager.toJdbc(Month.APRIL, null), is(4));
 	}
 
+	@Test
+	public void testWithCostom() throws ParseException {
```

Comment:
@HidekiSugimoto189 

This is a `testWithCustom` typo...

Reply:
@shout-star   

Thank you for the review. Fixed.